queries/orderbook/protocol_fees.sql

```diff
@@ -0,0 +1,105 @@
+-- Active: 1704993022452@@127.0.0.1@5432@postgres@public
+-- Active: 1704193756200@@cow-protocol-db-read-replica.c5bze0gkehvb.eu-central-1.rds.amazonaws.com@5432@mainnet@public
```

Comment:
This will be removed. (It is some artifact of how vscode handles running queries.)